io_scene_hubs/components.py

```diff
@@ -18,6 +18,18 @@ class HubsComponentName(PropertyGroup):
 class HubsComponentList(PropertyGroup):
     items: CollectionProperty(type=HubsComponentName)
 
+class HubsComponentsExtensionProperties(bpy.types.PropertyGroup):
+    enabled: bpy.props.BoolProperty(
+        name="Export Hubs Components",
+        description='Include this extension in the exported glTF file.',
+        default=True
```

Comment:
I don't know if we should default this to true.

Reply:
It will only do anything if you actually have hubs components added, in which case I think it makes sense to include them. Easy enough to "save export settings" and turn it off if that's not the case for some reason.